dotnet/src/IntegrationTests/IntegrationTests.csproj

```diff
@@ -61,8 +61,8 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\Connectors\Connectors.MistralAI\Connectors.MistralAI.csproj" />
-    <ProjectReference Include="..\Agents\Core\Agents.Core.csproj" />
-    <ProjectReference Include="..\Agents\OpenAI\Agents.OpenAI.csproj" />
+    <!--<ProjectReference Include="..\Agents\Core\Agents.Core.csproj" />
+    <ProjectReference Include="..\Agents\OpenAI\Agents.OpenAI.csproj" />-->
```

Comment:
Is this correct?

Reply:
This is merged through from the AI connector migration until they can consolidate / the V1 and V2 integration projects.

Reply:
Update - Now that the V1 connector has been removed and the dependency updated here, I think we can move forward.  Updating.